src/main/java/com/fabriccommunity/spookytime/common/SpookyBlocks.java

```diff
@@ -11,6 +13,7 @@
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemGroup;
 import net.minecraft.sound.BlockSoundGroup;
+import net.minecraft.util.Rarity;
```

Comment:
*cue 100 fabric community members arguing about one import, even though it's 100% necessary*

Reply:
But tbh what does Rarity do

Reply:
Does it mess with the coloured names

Reply:
Yes, rarities are how vanilla handles coloured names. See https://minecraft.gamepedia.com/Rarity